### ISSUE TITLE
Review Information in Requirements

### ISSUE THREAD

**Issue by Oglopf** [OSC/ood-documentation#928]
A discourse came in asking about the 5 minute PUN cleanup they read about in the requirements section:
https://osc.github.io/ood-documentation/latest/requirements.html#hardware-requirements

There's a line that says:
"The Passenger apps that make up the core of OnDemand (that NGINX is configured with), are each killed after a short period (5 minutes) of inactivity from the user"

I'm pretty sure that is not correct, and if it is I've never had my PUN cleaned up after 5 minutes of inactivity so I'd wonder what our setup is. Either way, this needs to be addressed and corrected or ensure it's correct.

**Comment by johrstrom** [comment]
> "The Passenger apps that make up the core of OnDemand (that NGINX is configured with), are each killed after a short period (5 minutes) of inactivity from the user"

That's correct, it comes from `passenger_pool_idle_time` in https://osc.github.io/ood-documentation/latest/reference/files/nginx-stage-yml.html

You may be able to tell by the load times. After that 5 minutes, the dashboard will have stopped. You hit the page and Nginx will just boot it up again.  You should be able to see messages around the same in `/var/log/ondemand-nginx/$USER/error.log` about passenger checking to see if there are any apps it should shutdown.
